[DOC] Update Management navigation interface in dashboards for UX revisions and add index management tutorial (#4550)
opensearch-project/documentation-website: _dashboards/im-dashboards/notifications.md

- Clone
- Force merge

To get started, under the OpenSearch Dashboards main menu, select **Management** > **Index Management**. Under **Index Management**, select **Notification settings**. 

You can choose to be notified when the operation has completed or failed. Additionally, you can select the notification channels for this notification, as shown in the following image.

![Default notification settings]({{site.url}}{{site.baseurl}}/images/admin-ui-index/notifications.png)

If you don't have permission to view notification settings, you cannot view the default settings. 
{: .note}

## Configuring notification settings for an individual operation

> You can view default notification settings when you perform an indexing operation, along with set up additional notifications. For example, if you want to configure an additional notification for a reindex operation, perform the following steps:

Reviewer: ```suggestion
You can view default notification settings when you perform an indexing operation as well as set up additional notifications. For example, if you want to configure an additional notification for a reindex operation, perform the following steps:
```

1. Select a channel from the **Notification channels** dropdown list. If you want to configure a new notification channel, select **Manage channels**.
    

>     To configure a new notification channel, confirm the `dashboards-notification` plugin is enabled in OpenSearch Dashboards. 

Reviewer: ```suggestion
    To configure a new notification channel, confirm that the `dashboards-notification` plugin is enabled in OpenSearch Dashboards. 
```
